test/docs.js

```diff
@@ -111,10 +111,17 @@ describe('documentation', function() {
       global: global
     };
 
+    function FakeExpress() {
+      return {
+        get: function() {}
```

Comment:
I'm not trying to be a stickler, but we've always strictly followed the "no example code untested" rule. That being said, this particular piece of code would be very easy to smoke test, so I don't see why we don't.

e.g.

```js
get: function(route, callback) {
  callback({ query: {} }, {});
}
```